doc: Fix Matter bridge application overview (#14307)
nrfconnect/sdk-nrf: applications/matter_bridge/README.rst

The Matter bridge device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power 802.11ax (Wi-Fi 6) network.

> Currently the Matter bridge application supports the following types of *bridged devices*:
> * Bluetooth® LE peripheral devices - devices that operate on real data (e.g. sensor measurements, lighting state)

Reviewer: ```suggestion
Currently the Matter bridge application supports the following types of *bridged devices*:

* Bluetooth® LE peripheral devices - Devices that operate on real data (such as sensor measurements or lighting state).
```

> * Simulated devices - devices that interact with the bridge by using data fabricated programmatically by the software

Reviewer: ```suggestion
* Simulated devices - Devices that interact with the bridge by using data fabricated programmatically by the software.
```

> Note that, apart from already implemented *bridged devices*, the Matter bridge architecture allows adding support for other devices that run over diverse connectivity technologies (e.g. Zigbee, Bluetooth® Mesh).

Reviewer: ```suggestion
Note that, in addition to already implemented *bridged devices*, the Matter bridge architecture allows you to add support for other devices that run over diverse connectivity technologies, such as Zigbee or Bluetooth® Mesh.
```
